build.sh

```diff
@@ -57,7 +57,13 @@ check_prereqs()
     hash cmake 2>/dev/null || { echo >&2 "Please install cmake before running this script"; exit 1; }
     
     # Check for clang
-    hash clang 2>/dev/null || { echo >&2 "Please install clang before running this script"; exit 1; }
+    OS=`uname`
+    if [ $OS = "Linux" ]; then
```

Comment:
so what would you suggest we do here ? check for both clang and clang-* for linux?.